cassandra-analytics-sidecar-client/src/main/java/org/apache/cassandra/clients/Sidecar.java

```diff
@@ -32,19 +32,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import o.a.c.sidecar.client.shaded.common.response.NodeSettings;
-import o.a.c.sidecar.client.shaded.io.vertx.core.Vertx;
-import o.a.c.sidecar.client.shaded.io.vertx.core.VertxOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.VertxOptions;
+import org.apache.cassandra.sidecar.common.client.SidecarInstance;
+import org.apache.cassandra.sidecar.common.client.SidecarInstanceImpl;
+import org.apache.cassandra.sidecar.common.response.NodeSettings;
+import org.apache.cassandra.sidecar.vertxclient.VertxHttpClient;
+import org.apache.cassandra.sidecar.vertxclient.VertxRequestExecutor;
```

Comment:
I think we still want to shade the sidecar client subproject and reference to the shadow jar. The idea is to embedding the sidecar client as a whole (shadow jar) w/o conflicting with any existing dependencies in analytics and spark.